rules/macos/persistence_loginwindow_plist_modification.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or modification of the Loin Window property list. Starting in Mac OS X 10.7 (Lion), users can
+specify certain applications to be re-opened when a user reboots their machine.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Persistence via Login Hook"
```

Comment:
```suggestion
name = "Potential Persistence via Login Hook"
note = "Starting in Mac OS X 10.7 (Lion), users can specify certain applications to be re-opened when a user reboots their machine. This can be abused to establish or maintain persistence on a compromised system."
```